src/pages/home/sidebar/OptionRow.js

```diff
@@ -63,6 +64,12 @@ const propTypes = {
     /** Whether to disable the interactivity of this row */
     disableRowInteractivity: PropTypes.bool,
 
+    /** Addtional styles to be passed to displayNameStyle */
+    textStyles: stylePropTypes,
+
+    /** Addtional styles to be passed to alternateTextStyle  */
+    alternateTextStyles: stylePropTypes,
+
```

Comment:
I think you missed the last comment because of resolved thread. 

> +1. Please do not mark comments resolved until there is an indication for the same.

It was this. 

--- 

I suggested using the styles on the issue but I feel that it would be unnecessary by looking at the code. It would be easier to just use one flag rather than provide the same styles from each component. If we need different styles in the future, we can add the props then. For now. please do this. 

> I don't think that gray color will ever change for OptionRow anywhere so instead of these props use a single prop. Something like showDisableState and then add the proper styles inside OptionRow.

Reply:
Okay